prime-angular-frontend/src/app/shared/components/contact-information-form/contact-information-form.component.html

```diff
@@ -44,12 +44,15 @@
     <app-page-subheader>
       <ng-container appPageSubheaderTitle>Notification Information</ng-container>
       <ng-container appPageSubheaderSummary>
-        Provide an email address 
-        <ng-container *ngIf="showSmsPhone">
-          and phone number (optional) 
-        </ng-container>
-        that may be used for sending notifications from the PRIME
-        system.
+        <div *ngIf="!forPhsa else PhsaSummary">
+          Provide an email address and phone number (optional)
```

Comment:
Specific text can be projected into the component using `<ng-content></ng-content>` to keep this generic